server/services/store/sqlstore/public_methods.go

```diff
@@ -47,12 +48,12 @@ func (s *SQLStore) AddUpdateCategoryBoard(userID string, categoryID string, boar
 }
 
 func (s *SQLStore) CanSeeUser(seerID string, seenID string) (bool, error) {
-	return s.canSeeUser(s.db, seerID, seenID)
+	return false, errors.New("can see user not supported in focalboard, will fetch from mattermost") 
 
 }
 
 func (s *SQLStore) CleanUpSessions(expireTime int64) error {
```

Comment:
These session function should be deleted from here, from `mattermostauthlayer` as well as they are no longer needed anywhere in the app.

Reply:
Sure.

Reply:
Removed it.